gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/sds/SdsRequestBuilder.java

```diff
@@ -19,21 +19,23 @@
 public class SdsRequestBuilder {
 
     private static final String ROUTING_AND_READABILITY_ENDPOINT = "/Endpoint";
+    private static final String ROUTING_AND_READABILITY_DEVICE_ENDPOINT = "/Device";
```

Comment:
Because this is the device endpoint I don't think is should be called "Routing and reliability" the [docs](https://digital.nhs.uk/developer/api-catalogue/spine-directory-service-fhir#get-/Device) call it Get Accredited System Details